kube/services/argo-events/workflows/configmap.yaml

```diff
@@ -97,6 +97,8 @@ data:
     metadata:
       name: workflow-WORKFLOW_NAME
     spec:
+      amiSelector:
+        aws::ids: ami-0d74e7059940620d8
```

Comment:
This is wrong I think 
<img width="1150" alt="image" src="https://github.com/uc-cdis/cloud-automation/assets/55899496/28c5e1e5-7657-4100-893a-016061b84b4c">